{
  "id": "ch_CSY11CK7T_2024-10-28_1730147682.887949_thread",
  "type": "channel",
  "channel_name": "discuss-frontend",
  "conversation_type": "thread",
  "participants": [
    "Liam Callanan",
    "Aayush",
    "Serena"
  ],
  "messages": [
    {
      "sender": "Liam Callanan",
      "user_id": "U03BXSHAK0T",
      "message": "Hey, anyone else seeing issues with `converse-app.spec.ts - uploads, digitizes, deletes, and displays the files correctly and responds correctly to pdf and csv queries` ?\n```Timed out retrying after 15000ms: Expected to find element: button[aria-label=\"add file to scope\"], but never found it. Queried from:\n> cy.get([data-testid=\"converse-file-viewer-item\"], [object Object]).contains(netflix-titles).parent().parent()```\ncc <!subteam^S07855GV3GV>\n\nLink to replay (https://cloud.cypress.io/projects/cfhedy/runs/15176/overview/bc465e5c-fd02-405d-aa44-5de763e0edf1/replay?roarHideRunsWithDiffGroupsAndTags=1&ts=1730136905131.5&att=1)",
      "time": "13:34",
      "timestamp": "1730147682.887949",
      "is_reply": false
    },
    {
      "sender": "Serena",
      "user_id": "U01CZ3LBFU4",
      "message": "cc @Aayush have you seen this? is this expected?",
      "time": "13:46",
      "timestamp": "1730148385.877879",
      "is_reply": true
    },
    {
      "sender": "Aayush",
      "user_id": "U01594ZKULX",
      "message": "Is this on master? Can you try rebasing - this should be fixed in this PR https://github.com/instabase/instabase/pull/64434",
      "time": "21:08",
      "timestamp": "1730174891.072029",
      "is_reply": true
    }
  ],
  "metadata": {
    "channel_id": "CSY11CK7T",
    "channel_name": "discuss-frontend",
    "date_file": "2024-10-28.json",
    "message_count": 3,
    "start_time": "1730147682.887949",
    "end_time": "1730174891.072029",
    "is_thread": true
  }
}